packages/davinci-client/src/lib/node.slice.ts

```diff
@@ -253,7 +255,9 @@ export const nodeSlice = createSlice({
       // TODO: Improve type
       const newState = state as Draft<NextNode>;
 
-      newState.client.collectors = nodeCollectorReducer(newState.client.collectors, action);
+      const a = nodeCollectorReducer(newState.client.collectors, action);
+
+      newState.client.collectors = a;
```

Comment:
Did you mean to commit this?

Reply:
Ah you're right was debugging. will back out that part.